src/Config.php

```diff
@@ -56,7 +56,8 @@ class Config
         'include_raw_request_body',
         'local_vars_dump',
         'max_nesting_depth',
-        'max_items'
+        'max_items',
+        'minimum_level'
```

Comment:
I believe this option would only be used with the Monolog handler. I suggest changing this config option to `monolog_minimum_level`. What do you think?

Reply:
I think is better because this params is found and maybe people use this. You have a good test how it's work. And very long name `monolog_minimum_level`.

Reply:
My concern is with the possible confusion of what this config option might be. This is something that will be only used with Monolog handlers. If we don't make that crystal clear a lot of users might mistakenly assume the config option will apply even if they don't use Monolog. Does it make sense?

Reply:
@ArturMoczulski I thought and still reworked on your offer.  =)